tests/zfs-tests/cmd/libzfs_input_check/libzfs_input_check.c

```diff
@@ -706,9 +706,10 @@ test_redact(const char *snapshot1, const char *snapshot2)
 	nvlist_free(snapnv);
 	nvlist_free(required);
 
-	strncpy(bookmark, snapshot1, sizeof (bookmark) - 1);
+	strlcpy(bookmark, snapshot1, sizeof (bookmark));
 	*strchr(bookmark, '@') = '\0';
-	strncat(bookmark, "#testbookmark", sizeof (bookmark));
+	strncat(bookmark, "#testbookmark", sizeof (bookmark) -
```

Comment:
Since we're switching to strlcpy() above, how about strlcat() for consistency?

Reply:
Updated.
@behlendorf Please merge sooner. It's been broken on recent Fedora for the past few days.